skyfield/starlib.py

```diff
@@ -40,7 +41,7 @@ class Star(object):
 
     def __init__(self, ra=None, dec=None, ra_hours=None, dec_degrees=None,
                  ra_mas_per_year=0.0, dec_mas_per_year=0.0,
-                 parallax_mas=0.0, radial_km_per_s=0.0, names=()):
+                 parallax_mas=0.0, radial_km_per_s=0.0, epoch=T0, names=()):
```

Comment:
Oh, there's one final adjustment needed — I should have noticed before: `epoch` needs to go last, else any code that folks have written that supplies plain non-keyword arguments for a `Star` will break if we take the argument that was once `names` and use that same slot for `epoch` instead. Try moving `epoch=` to the end so that existing calls keep working. Thanks!

Reply:
Done.